docs/git-selective-fork-sync-v1.md

```
@@ -0,0 +1,271 @@
+
+*Example reference: Brandon’s test branch – [https://github.com/brandonmarshal/lsx-demo-theme/tree/fork-sync-test/logs](https://github.com/brandonmarshal/lsx-demo-theme/tree/fork-sync-test/logs)*
+
+---
+
+[Goal](#goal)
+
+[Prerequisites](#prerequisites)
+
+[Quick naming convention](#quick-naming-convention)
+
+[Step‑by‑step](#step‑by‑step)
```

markdownlint-cli2 (0.18.1)

Link fragments should be valid

(MD051, link-fragments)